### PR TITLE
Adds Launch4J Windows build configs

### PR DESCRIPTION
**Pull request by Marwolf**

### CODE REVIEW

File: assets/build_windows.xml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<launch4jConfig>
+  <dontWrapJar>true</dontWrapJar>
+  <headerType>gui</headerType>
+  <jar>rscplus.jar</jar>
+  <outfile>C:\Users\Wolf\Downloads\rscplus-windows\rscplus.exe</outfile>
```
Comment (Hubcapp) [inline assets/build_windows.xml:6]: This path shouldn't be in

File: assets/build_windows.xml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  <errTitle></errTitle>
+  <cmdLine></cmdLine>
+  <chdir>.</chdir>
+  <priority>normal</priority>
+  <downloadUrl>https://adoptopenjdk.net/?variant=openjdk8&amp;jvmVariant=hotspot</downloadUrl>
+  <supportUrl></supportUrl>
+  <stayAlive>false</stayAlive>
+  <restartOnCrash>false</restartOnCrash>
+  <manifest></manifest>
+  <icon>C:\Users\Wolf\Downloads\rscplus-windows\jre\rscplus.ico</icon>
```
Comment (Hubcapp) [inline assets/build_windows.xml:16]: path

File: assets/build_windows.xml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  <versionInfo>
+    <fileVersion>1.2.0.0</fileVersion>
+    <txtFileVersion>1.2</txtFileVersion>
+    <fileDescription>Windows Launcher</fileDescription>
+    <copyright>GNU General Public License v3.0</copyright>
+    <productVersion>1.2.0.0</productVersion>
+    <txtProductVersion>1.2</txtProductVersion>
+    <productName>RSC Plus</productName>
+    <companyName></companyName>
+    <internalName>RSC Plus</internalName>
```
Comment (Hubcapp) [inline assets/build_windows.xml:35]: We don't stylize it `RSC Plus`.

It's either rscplus, RSCPlus (where + is not allowed) or RSC+ (Preferred)

**Review by Hubcapp — CHANGES_REQUESTED** [review]

File: assets/build_windows.xml (hunk 1 above)
Comment (Marwolf) [reply assets/build_windows.xml]: Pushed a fix

File: assets/build_windows.xml (hunk 3 above)
Comment (Marwolf) [reply assets/build_windows.xml]: Set to RSC+

**Comment by Hubcapp** [comment]
Looks good, will test & include in next release. Thanks!

**Review by Hubcapp — APPROVED** [review]

**Pull request merged**